RFC-011-SWAP-Omni-Layer-basic.md

```diff
@@ -0,0 +1,320 @@
+ The order of the inputs and outputs MUST be as described.
+ The *assigned Bitcoin value* on the inputs SHOULD be as described.
+
+ | Inputs (assigned Bitcoin value)      | Outputs (assigned Bitcoin value)                     |
+ |:---                                  |:---                                                  |
+ | 1. `htlc_utxo` (`min_sat`)           | 1. `to_address` (`min_sat`)                          |
+ | 2. `change_utxo` (`init_change_sat`) | 2. `change_address` (`init_change_sat - mining_sat`) |
+ |                                      | 3. `OP_RETURN omni_data` (`0`)                       |
+
+The implementor MAY include output (2) `change_address` or MAY decide to omit it and transfer the Bitcoin change to `to_address`.
+if output (2) `change_address` is present then input (2) `change_utxo` MUST be included to ensure the Omni Layer Assets are sent to output (1) `to_address`.
+
+
+## Execution Phase
+
+The following section describes how both parties should interact with the Bitcoin blockchain during the [RFC003 execution phase](./RFC-003-SWAP-basic.md#execution-phase).
+
+
+### Deployment transaction
+
+At the start of the deployment stage, both parties compile the contract as described in the previous section.
+We will call this value `contract_script`.
+
+To deploy the Bitcoin HTLC, the *funder* must confirm an Omni Layer transaction on the relevant Bitcoin blockchain.
+The transaction must have the following properties:
+
+- It MUST be a `valid` Omni Layer transaction.
+- It MUST be simple send Omni Layer transaction Class C.
+- The simple send `amount` MUST be equal to the `quantity` parameter in the Omni Layer asset header.
+- It MUST have an output locked with either
+  - a Pay-To-Witness-Script-Hash (P2WSH) `scriptPubKey` derived from `contract_script` (See [BIP141](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki#specification) for how to construct the `scriptPubkey` from the `contract_script`).
+  - OR a Pay-To-Witness-Script-Hash nested in Pay-To-Script-Hash (P2WSH(P2SH)) `scriptPubKey` derived from `contract_script` (See [BIP141](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#p2wsh-nested-in-bip16-p2sh) for how to construct the `scriptPubkey` from the `contract_script`).
+- The simple send `recipient` MUST be confirmed as a correct HTLC address, as defined in [HTLC Address](#htlc-address).
+
+
+Both parties MAY watch the blockchain for the deployment transaction, they MUST consider both (1) and (2) addresses.
+
+Such party MUST check the validity of the transaction against Omni Layer Spec.
+Not all valid Bitcoin transactions containing Omni Data are valid Omni Layer transactions.
+
+For example, the following RPC call to omnicore can be used to verify the validity of a deployment transaction:
+- `txid`: the transaction id of the transaction (e.g. `72abc494e877d7e2491ffa6fea3694648aa75e6166e42d6381b04d41ca933fb5`)
```

Comment:
🙃**transaction** id of the **transaction** :D ... I'd be fine with completely leaving out this description as the name says everything already...   
```suggestion
- `txid`: the transaction id (e.g. `72abc494e877d7e2491ffa6fea3694648aa75e6166e42d6381b04d41ca933fb5`)
```